locale/de/about/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: about.hbs
+title: Über Node.js
+trademark: Trademark
+---
+# Über Node.js&reg;
+
+Als asynchrone, Event-basierte Laufzeitumgebung wurde Node speziell für die
+Entwicklung von skalierbaren Netzwerkanwendungen entworfen. Im nachfolgenden
+"Hallo Welt" Beispiel können viele Verbindungen gleichzeitig bearbeitet werden.
+Bei jeder neuen Verbindung wird die Rückruffunktion ausgeführt. Gibt es jedoch
```

Comment:
"Rückruffunktion" -> Ich würde hier beim technischen Begriff "Callback-Funktion" bleiben, das wird AFAIK auch so in der gängigen deutschsprachigen Node.js-Literatur so gehandhabt.